front-end/app/checkout/checkout.component.html

```diff
@@ -29,7 +29,7 @@
             </div>
 
             <h3>Order notes</h3>
-            <textarea class="form-control" rows="3" name="order_notes" [(ngModel)]="orderNotes"></textarea>
+            <textarea class="form-control" rows="3" name="order_notes" [(ngModel)]="orderNotes" (blur)="saveOrderNotes()" (focus)="loadOrderNotes()"></textarea>
```

Comment:
I don't think you need to load order notes on focusing. I don't think it's a requirement in the ticket.

Also it would make your implementation a lot simpler if the order notes was a property on the front end basket service.